packages/create-pantheon-decoupled-kit/src/utils/runESLint.ts

```diff
@@ -21,12 +31,24 @@ export const runESLint = (
 	} else {
 		command = getPkgManager.name;
 	}
-	try {
-		execSync(`${command} lint`, { cwd: answers.outDir, stdio: 'inherit' });
-	} catch (error) {
-		if (error instanceof Error) {
-			throw error;
+
+	const filePath = path.resolve(`${answers.outDir}/package.json`);
+	const script = JSON.parse(fs.readFileSync(filePath, 'utf8')) as Script;
```

Comment:
Can we be a bit more specific with the names here?
```suggestion
	const pkgPath = path.resolve(`${answers.outDir}/package.json`);
	const pkg = JSON.parse(fs.readFileSync(filePath, 'utf8'));
```

I'm wondering if it would be better to break this functionality out since we may want to reuse this pattern in the future. See https://github.com/Sitecore/jss/blob/dev/packages/create-sitecore-jss/src/common/utils/helpers.ts#L33 for an example of how this could look. That way we could have a more robust type for the `package.json`.

Reply:
When breaking this functionality out, is it alright to use the `any` type similar to the Sitecore example?